prusti-tests/tests/verify_partial/fail/issues/issue-729-1.rs

```diff
@@ -0,0 +1,33 @@
+// error-pattern: Precondition of function snap$__$TY$__struct$m_A$Snap$struct$m_A might not hold
+// ^^^^^^^^^^^^^^ this is an actual assertion, see:
+// <https://rustc-dev-guide.rust-lang.org/tests/adding.html#when-error-line-cannot-be-specified>
```

Comment:
I think this is not needed. We would get an error from the test suite it we ever inadvertently modify it.